app/javascript/components/applications/ApplicationPage.js

```diff
@@ -18,20 +18,44 @@ const buildActionLinkIfNecessary = (app, showAddBtn) => {
 
 const ApplicationPage = (props) => {
   const { application, showAddBtn } = props
-  const pageHeader = {
-    title: `Application ${application.name}`,
-    content: (<span>Name of Listing: <a href={appPaths.toListing(application.listing.id)}>{application.listing.name}</a></span>),
-    action: buildActionLinkIfNecessary(application, showAddBtn)
+
+  let pageHeader = {}
+  let tabSection = false
+
+  if (application.is_snapshot) {
+    pageHeader = {
+      title: `${application.name}: ${application.applicant.name}`,
+      breadcrumbs: [
+        { title: 'Lease Ups', link: appPaths.toLeaseUps() },
+        { title: application.listing.name, link: appPaths.toListingLeaseUps(application.listing.id) },
+        { title: application.name, link: '#' }
+      ]
+    }
+
+    tabSection = {
```

Comment:
so this means that if you navigate to application through the application tab at the very top, and open an application that's in lease-up, you'll get the breadcrumbs and the tabs, even if you didnt start off from the lease up section, right?

Reply:
yes